src/main/java/org/datadog/jmxfetch/AppConfig.java

```diff
@@ -207,6 +207,13 @@
     @Builder.Default
     private boolean targetDirectInstances = false;
 
+    /**
+     * Boolean setting to determine whether to ignore jvm_direct instances.
+     * If set to true, other instances will be ignored.
```

Comment:
This inline comment needs to be changed.

I think it'd be useful for future reference to explain the use cases where we want to enable the option (namely, when JMXFetch is embedded in a client app, typically for the java tracer, so that the client app's exit doesn't block on the termination of these internal threads)

Reply:
Totally agree, this is a left over after copying and pasting another property. Changing right away.